src/bionics_ui.cpp

```diff
@@ -73,7 +73,7 @@ void draw_bionics_titlebar( WINDOW *window, player *p, bionic_menu_mode mode )
 
 const auto separator = []( std::ostringstream &s )
 {
-    return s.tellp() != 0 ? ", " : "";
+    return (int)(s.tellp()) != 0 ? ", " : "";
```

Comment:
Style regression.

It should looks like this: ```return ( int )( s.tellp() ) != 0 ? ", " : "";``` (please pay your attention to whitespaces).